src/game/savegame.c

```diff
@@ -167,30 +167,27 @@ static void SaveGame_LoadPostProcess()
     g_Lara.LOT.target_box = NO_BOX;
 }
 
-void InitialiseStartInfo()
+void SaveGame_InitStartEndInfo()
```

Comment:
You changed this function name but `InitialiseStartInfo` is still called in `inventry.c` twice and `shell.c` once and it not longer exists.

Reply:
Fixed